types/write-file-atomically/write-file-atomically-tests.ts

```diff
@@ -0,0 +1,10 @@
+import * as writeFileAtomically from 'write-file-atomically';
```

Comment:
Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

Reply:
I see. Thanks for the link. Strange that neither the lint nor the testing caught it.

Should I be using `export default` in this case? Or is it reasonable to expect users to say `import writeFileAtomically = require("write-file-atomically");`?

Reply:
Does the module actually provide a `default` export, or is it a CommonJS module that you could only import through `require` ?